pypdf/_utils.py

```diff
@@ -581,10 +581,8 @@ def __eq__(self, other: object) -> bool:
     def __lt__(self, other: Any) -> bool:
         if not isinstance(other, Version):
             raise ValueError(f"Version cannot be compared against {type(other)}")
-        min_len = min(len(self.components), len(other.components))
-        for i in range(min_len):
-            self_value, self_suffix = self.components[i]
-            other_value, other_suffix = other.components[i]
+
+        for (self_value, self_suffix), (other_value, other_suffix) in zip(self.components, other.components):
```

Comment:
For me, this change harms readability. Maybe leave the unpacking out of the loop control?